src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = []; // need to use static propery instead
+
+Animal.deleteDead = function() {
+  for (let i = 0; i < Animal.alive.length; i++) {
+    if (Animal.alive[i].health === 0) {
+      Animal.alive.splice(i, 1);
+    }
+  }
```

Comment:
not a good idea to manipulate with array in this way, instead you can you filter method